docs/querying/aggregations.md

```diff
@@ -422,6 +389,121 @@ It is not possible to determine a priori how well this aggregator will behave fo
 
 For these reasons, we have deprecated this aggregator and recommend using the DataSketches Quantiles aggregator instead for new and existing use cases, although we will continue to support Approximate Histogram for backwards compatibility.
 
+
+## Expression aggregators
```

Comment:
Do we need to list both the expression aggregator and the JavaScript aggregator under Expression aggregators? Can we  remove line 393 and change line 395 to H2 (## Expression aggregator)? If so, you can add an H3 section called Examples and list all of the expression aggregator examples there. 
For example:

## Expression aggregator
### Examples
## JavaScript aggregator

Reply:
I'm just wondering is the JavaScript agg worth it's own section? It's not enabled by default due to security issues

Reply:
What about "Expression aggregations" for H2 and then "Expression aggregator" and "JavaScript aggregator"? So something like this:

H2 Expression aggregations
H3 Expression aggregator
H4 Examples
H3 JavaScript aggregator
H4 Example